### ISSUE TITLE
False positive `g_error_init` in gdbus codegen

### ISSUE THREAD

**Issue by swick** [bilelmoussaoui/gobject-lint#40]
```
/workspace/build/src/xdp-impl-dbus.c:38060:3
  ...
  static void
  xdp_dbus_impl_screen_cast_proxy_set_property_cb (GDBusProxy *proxy,
    GAsyncResult *res,
    gpointer      user_data)
  ...
    const _ExtendedGDBusPropertyInfo *info = user_data;
  >  GError *error;
    GVariant *_ret;
    error = NULL;
    _ret = g_dbus_proxy_call_finish (proxy, res, &error);
  warning: GError *error must be initialized to NULL
  rule: g_error_init
```

**Comment by bilelmoussaoui** [comment]
You should ignore generated files using `--ignore "./build/*"` or setting that in the config file
